clappr/clappr-ios
# Refactor media control

Goal
---
To refactor `MediaControl` code and adjust its behaviour.

How to Test
---
1. Interact with the Media Control and check if it respects the following rules:

    - Every transition is animated, including showing/hiding drawers
    - It appears at the beggining of the video after the first play
    - Has a inactivity time of 3 seconds
    - It automatically disappears after 0.3 seconds after a user interaction when `playing`
    - It does not automatically disappears after user interaction when `paused` / `idle`

2. Run tests
